tests/test_app/TestApp/Command/EventsCommand.php

```diff
@@ -0,0 +1,38 @@
+<?php
+declare(strict_types=1);
+
+namespace TestApp\Command;
+
+use Cake\Command\Command;
+use Cake\Console\Arguments;
+use Cake\Console\ConsoleIo;
+use Cake\Event\EventInterface;
+
+class EventsCommand extends Command
+{
+    public static function getDescription(): string
+    {
+        return 'This is a command that uses events';
+    }
+
+    public function execute(Arguments $args, ConsoleIo $io): ?int
+    {
+        return null;
```

Comment:
```suggestion
    {
        $io->out('execute run');
    
        return null;
```

This would let us validate that `execute()` output will show up between before/after.